production.Dockerfile

```diff
@@ -7,7 +7,7 @@
 #
 # Build the frontend artifacts
 #
-FROM node:16.15-alpine3.14 AS frontend
+FROM node:16.15-alpine3.16 AS frontend
```

Comment:
Should we also update to 18 here? and for plugin-server?

Reply:
Yeah, I just wanted to have a 👍 from the ingestion team before doing so.

Reply:
You are right. I'll make a PR